website/docs/devDocs/Development.md

```diff
@@ -146,3 +146,33 @@ Checkstyle plugin may help to detect violations directly from the IDE.
    ```bash
    helm upgrade --set submarine.database.dev=true submarine ./helm-charts/submarine
    ```
+
+## Develop Submarine Website
+Submarine website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
+
+We store all the website content in markdown format in the `submarine/website/docs`. When committing a new patch to the `submarine` repo, Docusaurus will help us generate the `html` and `javascript` files and push them to  https://github.com/apache/submarine-site/tree/asf-site.
```

Comment:
Does Docusaurus run automatically? Or does the committer need to generate the doc and push it up?

Reply:
Docusaurus will run automatically in Github action when pushing the commit.